components/splitter/events.md

```diff
@@ -145,9 +145,94 @@ Try resizing any of the panes by dragging the splitbars
 }
 ````
 
+## SizeChanged
+
+`SizeChanged` is triggered when the `Size` parameter of the corresponding pane is changed.
+
+>caption Handle the SizeChanged event of a Splitter Pane
+
+````CSTHML
+@* Try resizing Pane 1 *@ 
+
+<div style="width: 500px; border: 1px solid red;">
+    <TelerikSplitter Width="100%"
+                     Height="200px">
+        <SplitterPanes>
+
+            <SplitterPane Size="200px" Collapsible="true">
+                <div>pane 0</div>
+            </SplitterPane>
+
+            <SplitterPane Size="@PaneSize" 
+                          SizeChanged="@SizeChangedHandler" 
+                          Collapsible="true">
+                <div>pane 1</div>
+            </SplitterPane>
+
+            <SplitterPane Collapsible="true">
+                <div>pane 2</div>
+            </SplitterPane>
+
+        </SplitterPanes>
+    </TelerikSplitter>
+</div>
+
+@code{
+    public string PaneSize { get; set; } = "250px";
+
+    void SizeChangedHandler(string size)
+    {
+        PaneSize = size;
+        Console.WriteLine("Pane 1 size was changed. Current size: " + PaneSize);
+    }
+}
+````
+
+
+## CollapsedChanged
+
+`CollapsedChanged` is triggered when the `Collapsed` parameter of the corresponding pane is changed.
```

Comment:
The same review as in the `SizeChanged` event

Reply:
Updated